crates/re_renderer/src/renderer/point_cloud.rs

```diff
@@ -364,14 +424,47 @@ impl PointCloudDrawData {
 }
 
 pub struct PointCloudRenderer {
-    render_pipeline: GpuRenderPipelineHandle,
+    render_pipeline_color: GpuRenderPipelineHandle,
+    render_pipeline_outline_mask: GpuRenderPipelineHandle,
     bind_group_layout_all_points: GpuBindGroupLayoutHandle,
     bind_group_layout_batch: GpuBindGroupLayoutHandle,
 }
 
+impl PointCloudRenderer {
+    fn create_point_cloud_batch(
+        &self,
+        ctx: &RenderContext,
+        label: DebugLabel,
+        uniform_buffer_binding: BindGroupEntry,
+        vertex_range: Range<u32>,
+        active_phases: EnumSet<DrawPhase>,
+    ) -> PointCloudBatch {
+        // TODO(andreas): There should be only a single bindgroup with dynamic indices here.
```

Comment:
I only see a single bindgroup

Reply:
fixing comment. it is still one bind group per batch, could be just one